modules/xmpp/RtxModifier.js

```diff
@@ -0,0 +1,242 @@
+import { getLogger } from "jitsi-meet-logger";
+const logger = getLogger(__filename);
+var transform = require('sdp-transform');
+var SDPUtil = require("./SDPUtil");
+
+/**
+ * Begin helper functions
+ */
+/**
+ * Given a videoMLine, returns a list of the video
+ *  ssrcs (those used to actually send video, not
+ *  any associated secondary streams)
+ * @param videoMLine media line object from transform.parse
+ * @returns list of ssrcs (as strings)
+ */
+function getPrimaryVideoSsrcs (videoMLine) {
```

Comment:
It feels like those helper functions could be extracted to some utility class, so that they would be more visible for someone unfamiliar with RtcModifier. The same applies to the SdpConsistency. Something like SdpTransformUtil that would gather helpers for operating with the 'sdp-transform' lib.